targets/unity-v2/Testing/Tests/Server/ServerInstanceApiTests.cs

```diff
@@ -0,0 +1,276 @@
+#if ENABLE_PLAYFABSERVER_API
+using PlayFab.ServerModels;
+using PlayFab.Json;
+using System;
+
+namespace PlayFab.UUnit
+{
+    public class ServerInstanceApiTests : UUnitTestCase
+    {
+        private const string FakePlayFabId = "1337"; // A real playfabId here would be nice, but without a client login, it's hard to get one
+
+        private static int SuccessfulLoginCount = 0, SuccessfulGetAllSegmentsCount = 0, UnsuccessfulGetAllSegmentsCount = 0, ParallelRequestSuccessfuCount = 0, ParallelRequestUnsuccessfuCount = 0;
+
+        private static TestTitleDataLoader.TestTitleData testTitleData;
+
+        public override void SetUp(UUnitTestContext testContext)
+        {
+            testTitleData = TestTitleDataLoader.LoadTestTitleData();
+        }
+
+        public override void Tick(UUnitTestContext testContext)
+        {
+            // No async work needed
+        }
+
+        private void SharedErrorCallback(PlayFabError error)
+        {
+            // This error was not expected.  Report it and fail.
+            ((UUnitTestContext)error.CustomData).Fail(error.GenerateErrorReport());
+        }
+
+        /// <summary>
+        /// SERVER API
+        /// Multiple instances of the same API class can be created
+        /// </summary>
+        [UUnitTest]
+        public void CreateMultipleServerInstance(UUnitTestContext testContext)
+        {
+            PlayFabApiSettings settings = new PlayFabApiSettings();
+            settings.TitleId = PlayFabSettings.TitleId;
```

Comment:
It would be more appropriate for:
```settings.TitleId = testTitleData.TitleId;```
Theoretically any titleId could be in PlayFabSettings, but testTitleData.TitleId is the one we're explicitly trying to use.